fml/backtrace.cc

```diff
@@ -4,27 +4,32 @@
 
 #include "flutter/fml/backtrace.h"
 
-#include <cxxabi.h>
-#include <dlfcn.h>
-#include <execinfo.h>
-
 #include <csignal>
 #include <sstream>
 
-#if FML_OS_WIN
-#include <crtdbg.h>
-#include <debugapi.h>
-#endif
-
+#include "flutter/fml/build_config.h"
 #include "flutter/fml/logging.h"
 #include "flutter/fml/paths.h"
 #include "third_party/abseil-cpp/absl/debugging/symbolize.h"
 
+#ifdef FML_OS_WIN
+#include <Windows.h>
+#include <crtdbg.h>
+#include <debugapi.h>
+#else  // FML_OS_WIN
+#include <cxxabi.h>
+#include <dlfcn.h>
+#include <execinfo.h>
+#endif  // FML_OS_WIN
+
 namespace fml {
 
 static std::string kKUnknownFrameName = "Unknown";
 
 static std::string DemangleSymbolName(const std::string& mangled) {
```

Comment:
Acoording to [docs](https://github.com/abseil/abseil-cpp/blob/ab2e2c4f6062999afaf960759dfccb77f350c702/absl/debugging/symbolize.h#L78), the `absl::Symbolize` is already demangling. Can we remove this function for all platforms?

Reply:
Looking at blame history, it looks like `DemangleSymbolName` was needed by [the original implementation](https://github.com/moko256/flutter_engine/commit/7c2d975b5556c0739f98448c994f05d53e6e9c41) but should've been removed when [we moved to abseil's implementation](https://github.com/moko256/flutter_engine/commit/f4bd18b9f8a3620d64808934032b26079cd46969). @chinmaygarde @jason-simmons Is that correct?

FWIW, `absl::Symbolize` definitely demangles on Windows (referred to as [undecorating by MSVC](https://docs.microsoft.com/en-us/cpp/build/reference/decorated-names?view=msvc-170)):

1. In [initialization it sets](https://github.com/abseil/abseil-cpp/blob/ab2e2c4f6062999afaf960759dfccb77f350c702/absl/debugging/symbolize_win32.inc#L47) the [`SYMOPT_UNDNAME` option](https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/symbol-options#symopt_undname)
2. [`Symbolize` calls `SymFromAddr`](https://github.com/abseil/abseil-cpp/blob/ab2e2c4f6062999afaf960759dfccb77f350c702/absl/debugging/symbolize_win32.inc#L65), which does the demangling.